README.md

```diff
@@ -1,6 +1,41 @@
 # helm-operator
 
-[![Build Status](https://github.com/joelanford/helm-operator/workflows/CI/badge.svg?branch=master)
-[![Coverage Status](https://coveralls.io/repos/github/joelanford/helm-operator/badge.svg?branch=master)](https://coveralls.io/github/joelanford/helm-operator?branch=master)
+[![Build Status](https://github.com/stackrox/helm-operator/workflows/CI/badge.svg?branch=main)
 
 Experimental refactoring of the operator-framework's helm operator
+
+### Why a fork?
+
+As the helm-operator is an experimental refactoring and not actively maintained we started a fork to 
+further support [hybrid operators](https://github.com/operator-framework/operator-sdk/issues/670) based on Helm.
+
+This fork should used as a library and not is recommended to watch CustomResources by configured `watches`.
```

Comment:
I don't understand this sentence, or how "watching CustomResources by configured watches" relates to whether or not it's being used as a library

Reply:
Rephrased the whole paragraph.